raco/language/myrialang.py

```diff
@@ -806,6 +838,9 @@ def __repr__(self):
     def num_tuples(self):
         return self._num_tuples
 
+    def partitioning(self):
```

Comment:
My link above was unclear: the plan I was referring to was `expr` in
```
 sql_plan = cat.get_sql(expr)
```
I've made the change in the latest commit